main.tf

```diff
@@ -180,21 +180,23 @@ module "athene2" {
 }
 
 module "kpi" {
-  source = "github.com/serlo/infrastructure-modules-kpi.git//kpi?ref=v1.0.0"
+  source = "github.com/serlo/infrastructure-modules-kpi.git//kpi?ref=f83107579c97737c5e119b169085532940a7d4a9"
```

Comment:
Should be source = "github.com/serlo/infrastructure-modules-kpi.git//kpi?ref=v1.1.0"